src/main/scala/gen/gmos/GmosNLongslitD.scala

```diff
@@ -94,107 +171,164 @@ object GmosNLongslitD {
 
   }
 
-  def apply[F[_]: Sync](
+  /**
+   * Unique step configurations used to form an acquisition sequence. Steps
+   * `ccd2` and `slit` are missing exposure time which must be set based upon
+   * ITC calculations.
+   *
+   * @param ccd2 initial image with CCD2 ROI
+   * @param p10  image through the slit at offset p 10
+   * @param slit image through the slit
+   */
+  final case class AcquisitionSteps(
+    ccd2: Step.GmosN,
+    p10:  Step.GmosN,
+    slit: Step.GmosN
+  )
+
+  object AcquisitionSteps extends GmosNOps {
+
+    // Find the filter with the closest wavelength.
+    private def filter(mode: ObservingMode.Spectroscopy.GmosNorth): GmosNorthFilter =
+      GmosNorthFilter.allAcquisition.minBy { f =>
+        (mode.λ.toPicometers - f.wavelength.toPicometers).abs
```

Comment:
Do you need to convert to `picometers` if you only care about the diff?

Reply:
We don't define subtraction or `abs` on `Wavelength` and we don't support negative `Wavelength`s.  Wavelengths are stored in pm so this seemed like the easiest thing to do.